tests/test_html5lib_shim.py

```diff
@@ -80,3 +80,65 @@ def test_serializer(data, expected):
     serialized = serializer.render(walker(dom))
 
     assert serialized == expected
+
+
+@pytest.mark.parametrize('parser_args, data, expected', [
```

Comment:
This tests the specific issue here plus other interesting `BleachHTMLParser` behavior.